src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -2,9 +2,14 @@
 
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+
+        Set<Employee> sortedSet = new TreeSet<>(new EmployeeComparator());
+        sortedSet.addAll(employees);
+
+        return sortedSet;
```

Comment:
Please do not add redundant empty lines:
```suggestion
        Set<Employee> sortedSet = new TreeSet<>(new EmployeeComparator());
        sortedSet.addAll(employees);
        return sortedSet;
```

Reply:
fixed